### PR TITLE
fix(emul-18): Breadcrumbs

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## [EMUL-18: Breadcrumbs](https://fourkitchens.atlassian.net/browse/EMUL-18)

**This PR does the following:**

- Creates the "BackLink" for the breadcrumbs on blog pages
- Creates the "WithSidebar" template
- Refactors the "Layout" template to "FullWidth"
- Creates the initial "Individual Blog" story
- Initial work to update Blog post content

### Functional Testing:

- [x] Run `yarn storybook`
  - [x] Checkout the [BackLink component](http://localhost:6006/?path=/story/atoms-backlink--back-link)
  - [x] Checkout the new [WithSidebar](http://localhost:6006/?path=/story/templates-page--with-sidebar) template
  - [x] Verify the old "Layout" template is now [FullWidth](http://localhost:6006/?path=/story/templates-page--full-width)
  - [x] Verify the [Individual Blog](http://localhost:6006/?path=/story/pages-blogs--individual-blog) correctly uses the "WithSidebar" template
- [x] Run `yarn develop` and verify the Backlink shows on a blog post on [the Gatsby site](http://localhost:8000/blog/test-entry).

**Notes:**
- I made some tweaks to the "Signup" component to make it look correct in the sidebar. It's currently only showing the heading (no form or submit button). That work is happening in a separate PR, I just wanted something there for now.
- I left the old content commented out in the `blog-post.tsx` file for future reference. It'll all be deleted once the blog post is fully updated.

### CODE REVIEW

File: src/components/molecules/Hero/Hero.tsx [hunk 1]
```diff
@@ -3,12 +3,12 @@ import React, { FC } from 'react'
 import styles from './hero.module.css'
 
 export type HeroProps = {
-  heading: string
+  heading?: string
```
Comment (ryanhagerty) [inline src/components/molecules/Hero/Hero.tsx:6]: Is this some Typescript nullish check?

**Comment by ryanhagerty** [comment]
Code looks good, I see this on `develop` though (doesn't happen in Storybook):
Screen Shot 2021-07-26 at 1 42 15 PM

**Comment by ryanhagerty** [comment]
I still get the CaseSensitive bug with `Card` and `CardGrid`. I can confirm I have the latest commit. 

The good news is that develop works and the image fits.

**Comment by ModulesUnraveled** [comment]
@ryanhagerty I couldn't reproduce the CaseSensitive issue you mentioned. I completely deleted the folder and re-cloned it. You might be able to delete the cache, and try again: `rm -rf node_modules/.cache/` then restart storybook.

**Review by ryanhagerty — APPROVED** [review]
🥑

File: src/components/molecules/Hero/Hero.tsx (hunk 1 above)
Comment (ModulesUnraveled) [reply src/components/molecules/Hero/Hero.tsx:6]: It just indicates that the variable is optional

Comment (ryanhagerty) [reply src/components/molecules/Hero/Hero.tsx:6]: Oh so like optional chaining without the chain

**Pull request merged**